runtime/calamari/Cargo.toml

```diff
@@ -68,6 +69,9 @@ cumulus-primitives-core = { git = 'https://github.com/paritytech/cumulus.git', d
 cumulus-primitives-timestamp = { git = 'https://github.com/paritytech/cumulus.git', default-features = false, branch = "polkadot-v0.9.18" }
 cumulus-primitives-utility = { git = 'https://github.com/paritytech/cumulus.git', default-features = false, branch = "polkadot-v0.9.18" }
 parachain-info = { git = 'https://github.com/paritytech/cumulus.git', default-features = false, branch = "polkadot-v0.9.18" }
+# Nimbus Dependencies
+nimbus-primitives = { git = "https://github.com/manta-network/nimbus.git", branch = "manta-v0.9.18", default-features = false }
```

Comment:
Did you make some changes in this branch? Not sure why you use this branch instead of like `polkadot-v0.9.18`.

Reply:
yes. check this commit
https://github.com/Manta-Network/nimbus/commit/009dfaa9f6f7eba47d232d776cb807dc24276d93

It needs to be redone when #571 is in